client/batch.go

```diff
@@ -77,91 +77,109 @@ func (b *Batch) initResult(calls, numRows int, err error) {
 	b.Results = append(b.Results, r)
 }
 
-func (b *Batch) fillResults() error {
+func (b *Batch) fillResults(br *proto.BatchResponse, pErr *proto.Error) error {
 	offset := 0
 	for i := range b.Results {
 		result := &b.Results[i]
 
 		for k := 0; k < result.calls; k++ {
-			call := b.calls[offset+k]
+			args := b.reqs[offset+k]
 
+			var reply proto.Response
 			if result.Err == nil {
-				result.Err = call.Reply.Header().GoError()
+				// TODO(tschottdorf): always do this (no errors on individual requests).
```

Comment:
I think the comment is misguided. I was mixing this up with `proto.Response`. I wasn't planning on changing the way `Batch.Results` works.